international-brazilian-journal-of-urology.csl

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="minimal-two" demote-non-dropping-particle="never" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>International Brazilian Journal of Urology</title>
```

Comment:
I've asked them at https://www.facebook.com/intbrazjurol/ via a private message (that Facebook page might be the only place that uses the full title) whether they can give any guidance.